### PR TITLE
Add in the new network connection parameters as per 1.22 api version

### LINKED ISSUE #460: Additional Network Options with 1.10
The api changed from api version 1.21 (1.9) to 1.22 (1.10) and the NetworkConnectionOptions ( https://godoc.org/github.com/fsouza/go-dockerclient#NetworkConnectionOptions) needs an update in order to align with the new api

1.21
https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#connect-a-container-to-a-network

1.22
https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#connect-a-container-to-a-network

### PR DESCRIPTION
**Pull request by lyondhill**
Fixes #460

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ var (
 
 	apiVersion112, _ = NewAPIVersion("1.12")
 
-	apiVersion119, _ = NewAPIVersion("1.19")
+	apiVersion119, _ = NewAPIVersion("1.22")
```
Comment (fsouza) [inline client.go]: You should not change this value.

**Comment by fsouza** [comment]
Hi @lyondhill, thank you for contributing and sorry for the late response. Could you have a look at the build failure, please?

**Comment by nickvanw** [comment]
Hey @lyondhill! It looks like this is pretty stale - would you mind me picking this up and creating a new branch/pr to accomplish this? I'm on a go-dockerclient kick, there's a few things on my to do list and this is one of them.

**Comment by lyondhill** [comment]
O go ahead. I got pulled off the project Iw as using that library for and would get back to it eventually.
You go ahead ;)
thanks

**Comment by fsouza** [comment]
Closing, as #480 has been merged.

@lyondhill thanks for contributing in the first place, and @nickvanw thanks for taking it over.

**Pull request closed without merging**